File: 프로그래머스/0/120869. 외계어 사전/외계어 사전.py
def solution(spell, dic):
    
    answer = [0] * len(dic)
    
    for i, d in enumerate(dic):
        for s in spell:
            if d.count(s) == 1:
                answer[i] += 1
    
    for a in answer:
        if a == len(spell):
            return 1
    return 2

File: 프로그래머스/0/120869. 외계어 사전/외계어 사전.java
class Solution {
    public int solution(String[] spell, String[] dic) {
        int answer = 0;
        
        for (int i=0; i<dic.length; i++) {
            int count = 0;
            for (int j=0; j<spell.length; j++) {
                if (dic[i].contains(spell[j])) count += 1;
            }
            if (count == spell.length) return 1;
            
        }
        return 2;
    }
}